index.html

```diff
@@ -1,16 +1,123 @@
+
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Cute Cakes</title>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+    <main class="homepage-main">
```

Comment:
I would use ``<main>`` tag as as the main section of the webpage (which contains the main image, welcome texts, and 4 images). Header should be outside of the main tag (same level as main tag). So you'd have something like:
```
<body>
   <header></header>
   <main></main>
   <footer></footer>
</body>
```